spec/metanorma_spec.rb

```diff
@@ -169,10 +169,3 @@ def ruby(x)
   command "metanorma -v -t iso"
   its(:stdout) { is_expected.to match(/Metanorma::ISO \d/) }
 end
-
-RSpec.describe "gives version Metanorma-Cli" do
```

Comment:
@CAMOBAP795 why are we removing this test?

Reply:
good catch, I decided to move it to version_spec but forgot to commit it, now it's done

Reply:
Excellent, thanks!